platform/i18n/src/locales/en-US/Common.json

```diff
@@ -1,4 +1,5 @@
 {
+  "Close": "Close",
```

Comment:
Were these changes made in `Locize` and then pulled?

Reply:
Yes -- all ` i18n` changes in this PR were pulled from Locize.